python/satyaml/RUZAEVKA-390.yml

```diff
@@ -0,0 +1,25 @@
+name: RUZAEVKA-390
+alternative_names:
+  - RS44S
+norad: 61766
+data:
+  &tlm AX25 telemetry:
+    telemetry: ax25
+transmitters:
+  1k135 AX25 AFSK downlink:
+    frequency: 437.050e+6
+    modulation: AFSK
+    baudrate: 1143
+    af_carrier: 1810
+    deviation: 565
```

Comment:
Were this parameters measured or taken from documentation?

Reply:
This information was shared by EU1SAT via this post https://community.libre.space/t/soyuz-2-1b-fregat-vostochny-launch-2024-11-04-23-18-utc/11468/66